src/build-template.js

```diff
@@ -17,16 +16,37 @@ module.exports.renderTemplate = function (changelogTemplate, data, version) {
   return injectMetadata(compiledTemplate(data), version);
 };
 
-module.exports.saveChangelogToFile = function (filePath, renderedTemplate) {
-  const fileDescriptor = fs.openSync(filePath, "a+");
+module.exports.saveChangelogToFile = function (
+  filePath,
+  renderedTemplate,
+  startString
+) {
+  let oldData = null;
+  if (startString) {
+    const [oldDataBeforeStartString, oldDataAfterStartString] = fs
+      .readFileSync(filePath)
+      .toString()
+      .split(startString);
+
+    oldData = oldDataAfterStartString;
+  } else {
+    oldData = fs.readFileSync(filePath);
+  }
+  fs.truncate(filePath);
+
+  const stream = fs.createWriteStream(filePath, { flags: "a" });
```

Comment:
Shouldn't you first write `oldDataBeforeStartString` to the file?

Reply:
There should be no data before (it's an empty string). Otherwise it would not be the `startString`

Reply:
mmmhmm.... sure. I update the PR 👍